<launch>
    <node pkg="navigation" type="BNO_Interface.py" name="BNO_Interface" output="screen"></node>
    <node pkg="navigation" type="listener.py" name="listener" output="screen"></node>
</launch>
